Assets/HoloToolkit/SpatialUnderstanding/Prefabs/SpatialUnderstanding.prefab

```diff
@@ -79,4 +79,6 @@ MonoBehaviour:
   m_Name: 
   m_EditorClassIdentifier: 
   ImportMeshPeriod: 1
-  MeshMaterial: {fileID: 2100000, guid: ddc2ff62e97ce6d41991784c3aa8e233, type: 2}
+  meshMaterial: {fileID: 2100000, guid: ddc2ff62e97ce6d41991784c3aa8e233, type: 2}
```

Comment:
Changing MeshMaterial to meshMaterial is the only intentional change here. I'm guessing the other differences are just due to resaving the file and picking up other changes that affect serialization.